ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-evpn-vpn-import-pruning.cfg

```diff
@@ -70,11 +70,15 @@ router bgp 65101
    vrf TENANT_A_PROJECT01
       rd 192.168.255.3:11
       route-target import evpn 11:11
+      route-target import evpn rcf RT_IMPORT_AF_RCF()
       route-target export evpn 11:11
+      route-target export evpn rcf RT_EXPORT_AF_RCF()
       router-id 192.168.255.3
```

Comment:
To get the same order as EOS we have to iterate on same loop twice or may be append in different list then combine it. Below is the output from EOS
```
vrf TENANT_A_PROJECT01
      route-target import evpn 11:11
      route-target export evpn 11:11
      route-target import evpn rcf RT_IMPORT_AF_RCF()
      route-target export evpn rcf RT_EXPORT_AF_RCF()
      redistribute connected
      redistribute static rcf Router_BGP_Static()
```

Reply:
cc: @ClausHolbechArista

Reply:
please check the ordering for route-map. If both are done afterwards, then adjust the template to loop twice. If not, leave it as-is.

Reply:
ah no, we cannot in the minor release, so leave it, and we will have to revisit in 5.0 work.